components/vf-masthead/vf-masthead.scss

```diff
@@ -2,14 +2,45 @@ $vf-masthead__title-text--color: set-color(vf-color-white);
 
 // vf-masthead
 
-.vf-header .vf-masthead {
-  display: grid;
-  grid-template-columns: minmax(var(--page-grid-gap), auto) minmax(auto, 76.5em) minmax(var(--page-grid-gap), auto);
+
+.vf-inlay__header {
+  margin: 0 auto;
```

Comment:
Hello, old friend.

Reply:
Yeah. The more I think about it. The more I’m thinking that whole page doesn’t need a grid right now. And we can use `:before` when we need colour or image ‘bars’.